src/interfaces/IMetaMorpho.sol

```diff
@@ -157,8 +157,10 @@ interface IMetaMorphoBase {
     /// @notice Warning: Anyone can supply on behalf of the vault so the call to `updateWithdrawQueue` that expects a
     /// market to be empty can be griefed by a front-run. To circumvent this, the allocator can simply bundle a
     /// reallocation that withdraws max from this market with a call to `updateWithdrawQueue`.
-    /// @dev Warning: Removing a market with supply will decrease the fee accrued until the next deposit/withdrawal.
-    /// @dev Warning: updateWithdrawQueue is not idempotent. Submitting twice the same tx will change the queue twice.
+    /// @dev Warning: Removing a market with supply will decrease the fee accrued until one of the functions updating
+    /// `lastTotalAssets` is triggered (deposit/mint/withdraw/redeem/setFee/setFeeRecipient). Specifically, if the funds
+    /// removed from the market are larger than the fee accrued on the other markets, the fee accrued will be zero.
```

Comment:
```suggestion
    /// `lastTotalAssets` is triggered (deposit/mint/withdraw/redeem/setFee/setFeeRecipient).
```
I don't find it necessary and it sounds a bit weird (what are "funds removed" ?)